lib/Importer/Caffe2.cpp

```diff
@@ -391,9 +393,13 @@ void caffe2ModelLoader::loadWeights(caffe2::NetDef &net) {
       }
 
       auto dim = getShape(dict["shape"]);
+      if (dim.empty())
```

Comment:
does onnx have the same behavior?

Reply:
ONNX has zero-dim tensor with dim_size() = 0, and it is viewed as a scalar. In ONNX doc, the zero-dim is written like: "shape(A) = (2, 3, 4, 5), shape(B) = (,), i.e. B is a scalar ==> shape(result) = (2, 3, 4, 5)".